resources/docs/html_en_EN/fixturemanager.html

```diff
@@ -100,6 +100,13 @@ <H3>Controls</H3>
    Opens the <a href="fixturesremap.html">Fixtures remapping</a> window.
   </TD>
  </TR>
+ <TR>
```

Comment:
Docs in HTML form are no longer bundled and maintained.
Please submit a PR here
https://github.com/mcallegari/qlcplus-docs/tree/main/pages/03.fixture-manager

Reply:
Happy to take care of it if this eventually makes it into master.